### PR TITLE
Fixed push + Images to default

### PR DESCRIPTION
**Pull request by ludwikkazmierczak**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java [hunk 1]
```diff
@@ -143,13 +143,13 @@ private String getFormattedHostForGlobalUrl(EnvType env) {
     switch (env) {
       case DEV: {
         String devBoxOwner = this.env.split("-")[1];
-        return String.join(".", "www", devBoxOwner, getDomain(envType));
+        return String.join(".", "www", devBoxOwner, envType.getWikiaDomain());
       }
       case SANDBOX: {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java]: MAJOR Extract this nested code block into a method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1199)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 6 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

#### 5 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [UrlBuilder.java#L28](https://github.com/Wikia/selenium-tests/blob/970e1371d5d5b09287a9869fccca71736423aa82/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L28): Remove this unused private "UrlBuilder" constructor. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MINOR][MINOR] [UrlBuilder.java#L56](https://github.com/Wikia/selenium-tests/blob/970e1371d5d5b09287a9869fccca71736423aa82/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L56): Define a constant instead of duplicating this literal "Page name is missing" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ArticleFeaturesCRUDTestsUser.java#L50](https://github.com/Wikia/selenium-tests/blob/970e1371d5d5b09287a9869fccca71736423aa82/src/test/java/com/wikia/webdriver/testcases/desktop/articlecrudtests/ArticleFeaturesCRUDTestsUser.java#L50): Define a constant instead of duplicating this literal "image" 8 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ArticleSourceModeTests.java#L182](https://github.com/Wikia/selenium-tests/blob/970e1371d5d5b09287a9869fccca71736423aa82/src/test/java/com/wikia/webdriver/testcases/desktop/articlecrudtests/ArticleSourceModeTests.java#L182): Define a constant instead of duplicating this literal "image" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ArticleSourceModeTests.java#L191](https://github.com/Wikia/selenium-tests/blob/970e1371d5d5b09287a9869fccca71736423aa82/src/test/java/com/wikia/webdriver/testcases/desktop/articlecrudtests/ArticleSourceModeTests.java#L191): Define a constant instead of duplicating this literal "\n</gallery>" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/common/contentpatterns/PageContent.java [hunk 2]
```diff
@@ -51,16 +51,16 @@
...
-  public static final String FILERENAME = "Image003.jpg";
+  public static final String FILERENAME = "default_Image003.jpg";
 
-  public static final String[] LIST_OF_FILES = {"Image001.jpg", "Image002.jpg", "Image003.jpg",
-                                                "Image004.jpg", "Image005.jpg", "Image006.jpg",
-                                                "Image007.jpg", "Image008.jpg", "Image009.jpg",
+  public static final String[] LIST_OF_FILES = {"default_Image001.jpg", "default_Image002.jpg", "default_Image003.jpg",
+                                                "default_Image004.jpg", "default_Image005.jpg", "default_Image006.jpg",
+                                                "default_Image007.jpg", "default_Image008.jpg", "default_Image009.jpg",
                                                 "Image010.jpg"};
```
Comment (emadry) [inline src/test/java/com/wikia/webdriver/common/contentpatterns/PageContent.java]: Image010 is not changed

File: src/test/java/com/wikia/webdriver/testcases/desktop/articlecrudtests/ArticleFeaturesCRUDTestsUser.java [hunk 3]
```diff
@@ -139,9 +139,9 @@ public void modifySlideshow() {
   @Test(groups = {"ArticleFeaturesCRUDUser_004"})
   public void deleteSlideshow() {
     new ArticleContent().push(
-        "<gallery type=\"slideshow\">\nImage010.jpg\nImage009.jpg\nImage008.jpg\nImage007.jpg\n"
-        + "Image010.jpg\nImage009.jpg\nImage008.jpg\nImage007.jpg\nImage006.jpg\nImage005.jpg\n"
-        + "Image004.jpg\nImage003.jpg\n</gallery>");
+        "<gallery type=\"slideshow\">\nImage010.jpg\ndefault_Image009.jpg\ndefault_Image008.jpg\ndefault_Image007.jpg\n"
+        + "Image010.jpg\ndefault_Image009.jpg\ndefault_Image008.jpg\ndefault_Image007.jpg\ndefault_Image006.jpg\ndefault_Image005.jpg\n"
```
Comment (emadry) [inline src/test/java/com/wikia/webdriver/testcases/desktop/articlecrudtests/ArticleFeaturesCRUDTestsUser.java]: Also here it's still Image010 and in test above

File: src/test/java/com/wikia/webdriver/common/core/api/ArticleContent.java [hunk 4]
```diff
@@ -15,25 +16,38 @@
...
   private User user = User.STAFF;
   private String username;
 
+  public ArticleContent(){
+    if(Configuration.getForceFandomDomain()){
+      this.baseURL = UrlBuilder.createUrlBuilder()
+                         .getUrl()
+                         .replace(UrlBuilder.HTTP_PREFIX, UrlBuilder.HTTPS_PREFIX)
+                     + "/api.php";
+    }else {
```
Comment (emadry) [inline src/test/java/com/wikia/webdriver/common/core/api/ArticleContent.java]: Missing space

**Comment by WikiaTech** [comment]
SonarQube analysis reported 6 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 4 minor

Watch the comments in this conversation to review them.

#### 5 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [UrlBuilder.java#L28](https://github.com/Wikia/selenium-tests/blob/10fd76cdc287d70271aca625cc7bbd02f97dc5de/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L28): Remove this unused private "UrlBuilder" constructor. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MINOR][MINOR] [UrlBuilder.java#L56](https://github.com/Wikia/selenium-tests/blob/10fd76cdc287d70271aca625cc7bbd02f97dc5de/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L56): Define a constant instead of duplicating this literal "Page name is missing" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ArticleFeaturesCRUDTestsUser.java#L50](https://github.com/Wikia/selenium-tests/blob/10fd76cdc287d70271aca625cc7bbd02f97dc5de/src/test/java/com/wikia/webdriver/testcases/desktop/articlecrudtests/ArticleFeaturesCRUDTestsUser.java#L50): Define a constant instead of duplicating this literal "image" 8 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ArticleSourceModeTests.java#L182](https://github.com/Wikia/selenium-tests/blob/10fd76cdc287d70271aca625cc7bbd02f97dc5de/src/test/java/com/wikia/webdriver/testcases/desktop/articlecrudtests/ArticleSourceModeTests.java#L182): Define a constant instead of duplicating this literal "image" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [ArticleSourceModeTests.java#L191](https://github.com/Wikia/selenium-tests/blob/10fd76cdc287d70271aca625cc7bbd02f97dc5de/src/test/java/com/wikia/webdriver/testcases/desktop/articlecrudtests/ArticleSourceModeTests.java#L191): Define a constant instead of duplicating this literal "\n</gallery>" 3 times. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**